lib/routes/javtiful/actress.ts

```diff
@@ -0,0 +1,37 @@
+import { Route, Data, DataItem } from '@/types';
+import ofetch from '@/utils/ofetch';
+import { load } from 'cheerio';
+import { parseItems } from './utils';
+
+export const route: Route = {
+    path: '/actress/:id',
+    name: 'Actress',
+    maintainers: ['huanfe1'],
+    example: '/javtiful/actress/akari-tsumugi',
+    parameters: { id: 'Actress name' },
+    handler,
+    categories: ['multimedia'],
+};
+
+async function handler(ctx): Promise<Data> {
+    const { id } = ctx.req.param();
+    const html = await ofetch(`https://javtiful.com/actress/${id}`, {
+        method: 'get',
+        headers: {
+            'Accept-Language': 'zh-CN,zh;q=0.9',
+        },
+    });
```

Comment:
```suggestion
    const html = await ofetch(`https://javtiful.com/actress/${id}`);
```
This header does not change the language of the site returning